### DISCUSSION TITLE
Is this project still alive?

### DISCUSSION THREAD

**Discussion by aetaylor** [farridav/django-jazzmin#486] [General]
There doesn't seem to be a great deal of activity.

**Comment by rob-gonz** [comment]
About a year ago I saw the developer had Covid.  He responded that he had it pretty bad. I was waiting on an upgrade that he said was coming.....I haven't seen anything from him since. :(

**Comment by PacificGilly** [comment]
Thanks for you interest and support for this project. We've started working on this package a lot in the past few weeks and have released a new version to support Django 4.2. More changes are coming. Have a look at the [README.md](https://github.com/farridav/django-jazzmin?tab=readme-ov-file#project-status) as we've updated the project status. Feel free to help out where you can. Thanks.

**Comment by rob-gonz** [comment]
@PacificGilly @farridav 

This is fantastic to hear!  I can finally update an existing project and have the confidence to add this to a new one. The other options out there may have a slightly better feature set, but they look like an admin out of the 90's. 

You all made my week to hear this is back in business.

**Reply by farridav** [reply]
Do tell us about this slightly better feature set :)

For right now we will be focused on clearing the PR/Issue backlog, code quality, compatibility/stability etc 

But once we are on top of that, we will start looking at the roadmap for features, and hopefully freshen up the UI a little bit.
